HEADER

```diff
@@ -1,7 +1,14 @@
-Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+Copyright 2021 Pravega Authors. 
```

Comment:
As far as I know, we are still keeping year relegated to NOTICE file. NOTICE file was updated by Igor previously.

Reply:
That's correct, I'll remove it.